src/Illuminate/Foundation/Console/ServeCommand.php

```diff
@@ -31,6 +31,26 @@ class ServeCommand extends Command
      */
     protected $portOffset = 0;
 
+    /**
+     * The environment variables that are passed thru to the built-in PHP webserver.
+     *
+     * @var array
+     */
+    protected $passthruEnvVars;
+
+    /**
+     * Create a new serve command instance.
+     *
+     * @param  array  $passthruEnvVars
+     * @return void
+     */
+    public function __construct(array $passthruEnvVars = ['APP_ENV', 'LARAVEL_SAIL', 'PHP_CLI_SERVER_WORKERS'])
+    {
+        parent::__construct();
+
+        $this->passthruEnvVars = $passthruEnvVars;
+    }
```

Comment:
does this allow the user to remove the required ```APP_ENV```` and ```LARAVEL_SAIL``` environment variables? Originally those two were hard coded, presumably as they are required. 

This will allow the user to effectively remove those variables by not passing them into the allowed array. Perhaps have a default array and ```array_merge``` the incoming ```$passthruEnvVars```